aip/general/0122.md

```diff
@@ -283,9 +283,8 @@ other use cases, use a synonymous term if possible.
 
 ### Fields representing another resource
 
-When referencing a resource name for a different resource, the field **should**
-be of type `string` for the resource name, and the field name **should** be
-equivalent to the corresponding message's name in snake case.
+When a field represents another resource, the field **should**
+be of type `string` and accept the resource name of the other resource. The field name **should** be equivalent to the corresponding message's name in snake case.
```

Comment:
Formatting: Need to wrap this line at 80 chars

Reply:
done!